server/src/main/java/org/apache/druid/server/coordinator/ReplicationThrottler.java

```diff
@@ -161,10 +161,10 @@ public void resetLifetime(String tier)
 
     public List<String> getCurrentlyProcessingSegmentsAndHosts(String tier)
     {
-      Map<SegmentId, String> segments = currentlyProcessingSegments.get(tier);
-      List<String> retVal = new ArrayList<>();
-      segments.forEach((segmentId, serverId) -> retVal.add(StringUtils.format("%s ON %s", segmentId, serverId)));
-      return retVal;
+      ConcurrentMap<SegmentId, String> segments = currentlyProcessingSegments.get(tier);
+      List<String> segmentsAndHosts = new ArrayList<>();
+      segments.forEach((segmentId, serverId) -> segmentsAndHosts.add(segmentId + " ON " + serverId));
```

Comment:
I wonder why you didn't use `StringUtils.format`.

Reply:
Simple string concatenation should be faster because it doesn't involve parsing of the format.

Reply:
Got it.